qiskit/circuit/library/arithmetic/exact_reciprocal.py

```diff
@@ -31,14 +31,16 @@ def __init__(
         r"""
         Args:
             num_state_qubits: The number of qubits representing the value to invert.
-            scaling: Scaling factor of the reciprocal function, i.e. to compute
-             ::math:: `scaling / x`.
+            scaling: Scaling factor of the reciprocal function, i.e. to compute :math:`scaling / x`.
```

Comment:
This is probably gonna be rendered poorly, how about
```suggestion
            scaling: Scaling factor :math:`s` of the reciprocal function, i.e. to compute
                :math:`s / x`.

```

Reply:
Done in aa157bc.